app/Account.php

```diff
@@ -580,6 +603,76 @@ public function replaceGender(Gender $genderToDelete, Gender $genderToReplaceWit
         return true;
     }
 
+    /**
+     * Get if any account exists on the database.
+     *
+     * @return bool
+     */
+    public static function hasAny()
+    {
+        return DB::table('accounts')->count() > 0;
+    }
+
+    /**
+     * Create a new account and associate a new User.
+     *
+     * @param string $first_name
+     * @param string $last_name
+     * @param string $email
+     * @param string $password
+     * @return this
+     */
+    public static function createDefault($first_name, $last_name, $email, $password)
+    {
+        // create new account
+        $account = new self;
+        $account->api_key = str_random(30);
+        $account->created_at = Carbon::now();
```

Comment:
you should use `now()` helper - see #1051

Reply:
done